src/controller/ToDoController.java

```diff
@@ -27,12 +29,13 @@ private void showMenu() {
     System.out.println("2. Update an existing task");
     System.out.println("3. Delete an existing task");
     System.out.println("4. List all Tasks");
-    System.out.println("5. Exit");
+    System.out.println("5. Search Task By ID");
+    System.out.println("6. Exit");
   }
 
   private String getUserInput(String inputMessage) {
     System.out.print(inputMessage + " : ");
-    return System.console().readLine();
+    return new Scanner(System.in).nextLine();
```

Comment:
Why is this change done?

Reply:
`System.cosole().readLine();` this throws NPE while running in an IDE so I changed that line,  Reverted it back because its working from console.

Reply:
ok